src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -699,6 +724,7 @@ static grpc_error* cc_init_channel_elem(grpc_channel_element* elem,
   // Record enable_retries.
   arg = grpc_channel_args_find(args->channel_args, GRPC_ARG_ENABLE_RETRIES);
   chand->enable_retries = grpc_channel_arg_get_bool(arg, true);
+  chand->channelz_channel = nullptr;
```

Comment:
No need for this; `chand` is initialized to all zero.

Reply:
I think @soheilhy is working to change this invariant in the name of performance.

Mind if I keep to make his life easier?

Reply:
Yes, thank you for initialization it! I'm actually preparing the PR at the moment. :-)

Reply:
Okay, that's fine.

@soheilhy, I will be very interested in reviewing that PR.  Thanks for doing it!

Reply:
Absolutely, I'd really appreciate your time and feedback :-) ETA is (hopefully) tomorrow morning.